pkgs/applications/misc/logseq/default.nix

```diff
@@ -4,7 +4,7 @@
 , appimageTools
 , makeWrapper
 # graphs will not sync without matching upstream's major electron version
-, electron_27
+, electron_28
```

Comment:
Ideally pin version at `callPackage` (top-level in this case) to avoid messing with overrides.
Whenever versions change it shouldn't change override interface.

Reply:
Hello that's a good idea.. so I'm new as i understand we should define the electron version at the top.